### ISSUE TITLE
how to solve "ghost-like illusion" in outputs

### ISSUE THREAD

**Issue by sd2530615** [kijai/ComfyUI-HunyuanVideoWrapper#107]
it's like there are a ghost-like shadow or illusion that mimic the objects movements but delayd like 0.2s.

like the object is in at slowmotion . idk how to name it ,might be ghost.

**Comment by kijai** [comment]
Can't say I have seen that, can you show an example?

**Comment by sd2530615** [comment]
https://github.com/user-attachments/assets/ce898c25-cfa8-4bb3-9062-f2ae30db47ee



> Can't say I have seen that, can you show an example?

here is the example ,you can see that ghost near the girls especially near the arms

**Comment by kijai** [comment]
How many steps? Sometimes more helps with that, it's common issue with limbs in general, I think it's partly because of the fp8 quant.

**Comment by sd2530615** [comment]
![T}B (P5}0S177 PO2N(O_ H](https://github.com/user-attachments/assets/541abb24-42a2-4a43-97b5-940dffc1cc9e)

30 steps.the screenshot of workflow,also the video above embed the workflows.basically as the same as exmaple workflows.

ive checked the decode settings and it seems not likely causing by 2 tile size settings.

**Comment by sd2530615** [comment]
@kijai i found something interesting,  i use the same settings and seed of the examples above,  but revert wrapper to old version of "Commit 0665f91", it seems those ghosts are more or less gone. they might be still there but much more better.

https://github.com/user-attachments/assets/78504d0c-8542-4690-a5ab-51d32fbd504e

**Comment by sd2530615** [comment]
also turn the auto off,and setting the minsize to 256 causing the ghosts too. @kijai

**Comment by kijai** [comment]
Oh, right yeah, the temporal tile size wasn't working before, it was always 64 and the setting did nothign. As I fixed it I added the auto_tile size which still sets it to 64 in the background, like before, so auto on is the same as the old behaviour. If you want to change the tile size, put the temporal to 64 and only reduce the spatial if you run out of memory. If you have no memory issues, leave the auto_tile on.

**Comment by sd2530615** [comment]
> Oh, right yeah, the temporal tile size wasn't working before, it was always 64 and the setting did nothign. As I fixed it I added the auto_tile size which still sets it to 64 in the background, like before, so auto on is the same as the old behaviour. If you want to change the tile size, put the temporal to 64 and only reduce the spatial if you run out of memory. If you have no memory issues, leave the auto_tile on.

yes! leave temporal tiling sample size to 64,  everything if fixed ,thanks for you answering with patience.

**Comment by 4lt3r3go** [comment]
Ok, can we have a clear idea about these values so i share this info somewere else?
 
-recommended settins for best quality, for Vram consumption, compromises and so on.
Ty

![image](https://github.com/user-attachments/assets/5b6cc398-dea5-49be-b425-be9632622bfb)
